### DISCUSSION TITLE
关于挂载阿里云及OneDrive播放卡顿问题

### DISCUSSION THREAD

**Discussion by Treceluo** [AlistGo/alist#448] [Q&A] [answered]
作者大大你好，我想了解下alist挂载的网盘带宽速度是走的网盘服务商本身的（例如OneManager挂载OneDrive直链播放是以微软服务器带宽为主，测试potplayer在线播放约30MB/S，家宽300M），还是走的是搭建alist服务器的带宽，目前搭建alist的服务器用的是阿里的ecs，带宽1m，发现不管是挂载了阿里云盘还是OneDrive，直链播放只有20-30kb左右，即使点击下载，速度仍然只有30kb，

**Comment by xhofe** [comment, marked as answer]
OneDrive添加账号时有个web proxy 选项，打开走服务器，关闭走网盘本身。具体见：
https://alist-doc.nn.ci/docs/driver/base#proxy%E4%BB%A3%E7%90%86

**Reply by Treceluo** [reply]
好的哈，非常感谢
